pkg/opc/opc.go

```diff
@@ -336,3 +336,38 @@ func VerifyResourceListMatchesName(resourceType, name, namespace string) (string
 	}
 	return output, nil
 }
+
+func CheckInstalledVersion(binary string) {
```

Comment:
What is the purpose of this test in the ecosystem test? We already do this check. See line 99.